.github/PULL_REQUEST_TEMPLATES/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Merge Checklist
+
+<!-- Complete all tasks below to make sure that we can merge your pull request. -->
+
+- [ ] I have read the [contribution guide](doc/contributing.md)
+- [ ] I have added my info to [copying.md](copying.md) (only first time contributors)
+- [ ] I have run `make checkall` and fixed all mentioned problems
+
+
+### Description
+
+<!-- Describe the changes this pull request makes to the code base. -->
+
+
+### Related Issue
+
+<!-- If this pull request is related to an issue, replace LINK with the link to the GitHub issue. -->
+
+Resolves LINK
```

Comment:
people will probably ignore this - maybe it should be a comment on how to add related issues. and the relation shouldn't be an extra heading imo.

Reply:
Yeah I think I will remove this entirely and we just do this ourselves when we see the PR.